src/python/turicreate/toolkits/drawing_classifier/__init__.py

```diff
@@ -10,4 +10,4 @@
 from ..image_classifier._annotate import annotate, recover_annotation
 from . import util
 
-__all__ = ['create', 'DrawingClassifier', 'util', 'annotate', 'recover_annotation']
```

Comment:
Why don't we want `annotate` and `recover_annotation` here?

Reply:
Ah. I took it out because I was trying to see what was working or not. This has nothing to do with the fix. Will put it back. Thanks